app/goal_routes.py

```diff
@@ -0,0 +1,77 @@
+from flask import Blueprint, jsonify, make_response, abort, request
```

Comment:
👍🏾 These imports help a great deal when it comes to our request & response cycle